File: golang/excel_sheet_column_title.go
package main

// Space	: O(1)
// Time		: O(26 log n)

func convertToTitle(c int) string {
    ans := ""
    
    for c > 0 {
        c--
        ans = string(byte(65+(c % 26))) + ans
        c /= 26
    }
    
    return ans
}

File: python3/excel_sheet_column_title.py
# Space : O(1)
# Time  : O(26 log n)

class Solution:
    def convertToTitle(self, n: int) -> str:
        ans = ""

        while n > 0:
            n -= 1
            ans = chr(65+(n % 26)) + ans
            n //= 26

        return ans
